src/templates/en/2019/chapters/pwa.html

```diff
@@ -267,6 +264,29 @@ <h2 id="workbox">Workbox</h2>
         12.71% of them are using at least one of the Workbox libraries. This percentage is roughly consistent <br />
         across desktop and mobile, with a slightly lower percentage (11.46%) on mobile compared to desktop (14.36%).
       </p>
+      <h2 id="conclusion">Conclusion</h2>
+      <p>
+        The stats in this chapter show that PWAs are still only used by a small percentage of sites. However this relatively small usage<br />
+        is driven by the more popular sites which have a much larger share of traffic, and pages beyond the home page may use this more:<br />
+        we showed that 15% of page loads use a service workers. The advantages they give for <a href="./performance">performance</a> and<br />
+        greater control over <a href="./caching">caching</a> particularly for <a href="./mobile">mobile</a>.
+      </p>
+      <p>
+        PWAs have often been seen as Chrome-driven technology. Other browsers have made great strides recently to implement most of the<br />
+        underlying technologies, although first-class installability lags on some platforms. it's positive to see support becoming more<br />
+        widespread. <a href="https://twitter.com/firt">Maximiliano Firtman</a> does a great job of tracking this, including<br />
+        <a href="https://medium.com/@firt/iphone-11-ipados-and-ios-13-for-pwas-and-web-development-5d5d9071cc49">explaining Safari PWA support</a>.<br />
+        Apple doesn't use the term PWA much, and has<br />
+        <a href="https://developer.apple.com/news/?id=09062019b">explicitly stated that these HTML5 apps are best delivered outside of the App Store</a>.<br />
+        Microsoft went the opposite direction not only <a href="https://docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps/microsoft-store">encouraging PWAs in it's app store, but even automatically indexing any found when as it craweled the web</a>, though with it<br />
+        moving on from the Windows Phone platform the future of that is uncertain. Google has also<br />
+        <a href="https://medium.com/@firt/google-play-store-now-open-for-progressive-web-apps-ec6f3c6ff3cc">made moves to accepting PWAs in its app store</a>.
+      </p>
+      <p>
+        PWAs provide a path forward for developers who would prefer to build and release on the web instead of on native platforms and app stores.<br />
```

Comment:
```suggestion
        PWAs provide a path forward for developers who would prefer to build and release on the web instead of on native platforms.<br />
```

(Because I realize the previous bit is all about PWAs and app stores.)